File: backend/ws_wa_store_ccadmin/src/main/java/com/ccadmin/app/cash/model/dto/CurrentCashSessionDto.java
package com.ccadmin.app.cash.model.dto;

import com.ccadmin.app.cash.model.entity.CashRegisterEntity;
import com.ccadmin.app.cash.model.entity.CashSessionEntity;

public class CurrentCashSessionDto {

    public CashRegisterEntity CashRegister;
    public CashSessionEntity CashSession;
    public boolean IsOpen;

    public CurrentCashSessionDto(CashRegisterEntity cashRegister, CashSessionEntity cashSession) {
        this.CashRegister = cashRegister;
        this.CashSession = cashSession;
        this.IsOpen = cashSession != null;
    }
}

File: frontend/web_ccadmin_lt/src/app/enterprise/cash/model/dto/CurrentCashSessionDto.ts
import { CashRegisterEntity } from "../entity/CashRegisterEntity";
import { CashSessionEntity } from "../entity/extends AuditTableEntity";

export class CurrentCashSessionDto {
    public CashRegister: CashRegisterEntity | null;
    public CashSession: CashSessionEntity | null;
    public IsOpen: boolean;

    constructor() {
        this.CashRegister = null;
        this.CashSession = null;
        this.IsOpen = false;
    }
}
